csp-tech-radar.csv

```diff
@@ -17,7 +17,7 @@ Anemic REST,hold,techniques,TRUE,"Failure to properly model the domain as a set
 ELK Stack,adopt,tools,TRUE,"Supporting and developing distributed systems requires the ability to debug events that occur across many systems. Having the logs emitted from each system centralised in a single store that is easily searched is vital for this. The ELK (Elasticsearch, Logstash and Kibana) Stack is a mature open source platform for this task."
 Prometheus and Grafana,adopt,tools,TRUE,"For monitoring and altering of the health of applications in production, the ability to collect and store time series data is important. Prometheus and Grafana provide a set of mature tools to achieve this."
 Consul,trial,tools,TRUE,"Consul is a the service-discovery tool supporting both DNS- and HTTP-based discovery mechanisms. It goes beyond other discovery tools by providing customizable health checks for registered services, ensuring that unhealthy instances are marked accordingly. More tools have emerged to work with Consul to make it even more powerful. Consul Template enables configuration files to be populated with information from Consul, making things like client-side load balancing using mod_proxy much easier. In the world of Docker, registrator can automatically register Docker containers as they appear with Consul with extremely little effort, making it much easier to manage container-based setups. You should still think long and hard about whether you need a tool like this or whether something simpler will do, but if you decide you need service discovery, you won't go far wrong with Consul."
-Pact Foundation,trial,tools,TRUE,"A cross platform testing tool framework used for Consumer Driven Contract Testing. It generates JSON contracts via unit tests which can then be exchanged with the provider. It involves minimal setup for both Provider and Consumer and has well written documentation and resources dedicated to it. Its power lies in its simplicity."
+Pact Foundation,trial,tools,TRUE,A cross platform testing tool framework used for Consumer Driven Contract Testing. It generates JSON contracts via unit tests which can then be exchanged with the provider. It involves minimal setup for both Provider and Consumer and has well written documentation and resources dedicated to it. Its power lies in its simplicity.
```

Comment:
Any reason why you removed the quotes from this entry description field?

Reply:
Doh. I didn't do that on purpose! Let me resolve.